enterprise/coderd/features_internal_test.go

```diff
@@ -1,545 +0,0 @@
-	})
-}
-
-func requestEntitlements(t *testing.T, uut features.Service) codersdk.Entitlements {
-	t.Helper()
-	r := httptest.NewRequest("GET", "https://example.com/api/v2/entitlements", nil)
-	rw := httptest.NewRecorder()
-	uut.EntitlementsAPI(rw, r)
-	resp := rw.Result()
-	defer resp.Body.Close()
-	assert.Equal(t, http.StatusOK, resp.StatusCode)
-	dec := json.NewDecoder(resp.Body)
-	var result codersdk.Entitlements
-	err := dec.Decode(&result)
-	require.NoError(t, err)
-	return result
-}
-
-func putLicense(
-	ctx context.Context, t *testing.T, db database.Store,
-	k ed25519.PrivateKey, keyID string, userLimit int64,
-	timeToGrace, timeToExpire time.Duration,
-) database.License {
-	t.Helper()
-	c := &Claims{
-		RegisteredClaims: jwt.RegisteredClaims{
-			Issuer:    "test@testing.test",
-			ExpiresAt: jwt.NewNumericDate(time.Now().Add(timeToExpire)),
-			NotBefore: jwt.NewNumericDate(time.Now().Add(-time.Minute)),
-			IssuedAt:  jwt.NewNumericDate(time.Now().Add(-time.Minute)),
-		},
-		LicenseExpires: jwt.NewNumericDate(time.Now().Add(timeToGrace)),
-		Version:        CurrentVersion,
-		Features: Features{
-			UserLimit: userLimit,
-			AuditLog:  1,
-		},
-	}
-	j, err := makeLicense(c, k, keyID)
-	require.NoError(t, err)
-	l, err := db.InsertLicense(ctx, database.InsertLicenseParams{
-		UploadedAt: c.IssuedAt.Time,
-		JWT:        j,
-		Exp:        c.ExpiresAt.Time,
-	})
-	require.NoError(t, err)
-	return l
-}
-
-func userLimitIs(fs *featuresService, limit int64) func(context.Context) bool {
-	return func(_ context.Context) bool {
-		fs.mu.RLock()
-		defer fs.mu.RUnlock()
-		return fs.entitlements.activeUsers.limit == limit
-	}
-}
-
-func TestFeaturesServiceGet(t *testing.T) {
-	t.Parallel()
-	logger := slogtest.Make(t, nil)
-
-	// Note that these are not actually used because we don't run the syncEntitlements
-	// routine in this test.
-	pubsub := database.NewPubsubInMemory()
-	pub, _, err := ed25519.GenerateKey(rand.Reader)
-	require.NoError(t, err)
-	keyID := "testing"
-	db := databasefake.New()
-
-	t.Run("AuditorOff", func(t *testing.T) {
-		t.Parallel()
-		uut := &featuresService{
-			logger:      logger,
-			database:    db,
-			pubsub:      pubsub,
-			keys:        map[string]ed25519.PublicKey{keyID: pub},
-			enablements: Enablements{AuditLogs: true},
-			enabledImplementations: agplCoderd.FeatureInterfaces{
-				Auditor: audit.NewAuditor(audit.DefaultFilter),
-			},
-			entitlements: entitlements{
-				hasLicense: false,
-				activeUsers: numericalEntitlement{
-					entitlement{notEntitled},
-					entitlementLimit{
-						unlimited: true,
-					},
-				},
-				auditLogs: entitlement{notEntitled},
-			},
-		}
-		target := struct {
-			Auditor agplAudit.Auditor
-		}{}
-		err := uut.Get(&target)
-		require.NoError(t, err)
-		assert.NotNil(t, target.Auditor)
-		nop := agplAudit.NewNop()
-		assert.Equal(t, reflect.ValueOf(nop).Type(), reflect.ValueOf(target.Auditor).Type())
-	})
-
-	t.Run("AuditorOn", func(t *testing.T) {
```

Comment:
Where did these test cases go?  We need tests that licensing the coderd actually turns on audit logging.

Reply:
Same comment as above.